tests/unit/drtsans/tof/eqsans/test_load.py

```diff
@@ -204,6 +204,54 @@ def test_generic_load_and_split(datarepo_dir, clean_workspace):
     assert SampleLogs(filtered_ws.getItem(2)).slice_start.units == "C"
 
 
+@pytest.mark.datarepo
+def test_load_and_split_periodic_timeslice_loadoptions(datarepo_dir, clean_workspace):
+    """Test the interaction between periodic time slicing and the loadOptions FilterByTimeStart and FilterByTimeStop"""
+    # load the run
+    unfiltered_ws = generic_load_and_split(
+        "EQSANS_104088.nxs.h5",
+        data_dir=datarepo_dir.eqsans,
+        time_interval=10.0,
+        time_period=30.0,
+    )
+    [clean_workspace(_ws) for _ws in list(unfiltered_ws)]
+
+    assert unfiltered_ws.size() == 3
+
+    assert SampleLogs(unfiltered_ws.getItem(0)).duration.value == pytest.approx(100.0)
+    assert SampleLogs(unfiltered_ws.getItem(1)).duration.value == pytest.approx(99.0459)
+    assert SampleLogs(unfiltered_ws.getItem(2)).duration.value == pytest.approx(90.0)
+
+    assert unfiltered_ws.getItem(0).getNumberEvents() == 473643
+    assert unfiltered_ws.getItem(1).getNumberEvents() == 468791
+    assert unfiltered_ws.getItem(2).getNumberEvents() == 426051
```

Comment:
same here. If we add the three event counts, does it amount to the number of events in run `EQSANS_104088.nxs.h5`?